1 - identify organisations in the corpus/gather_texts.ipynb

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #43.                addon = addon.replace("format title author subject keywords creator producer creationDate modDate trapped encryption id", "")
</pre><p>Why do you add these tokens to the output text file? Maybe add a comment why this is necessary!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/disaster-capitalism/named-entities/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Anomalous text left behind from scraping at the start of each document. I moved this "processing" step to the preprocessing.py file.